src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json

```diff
@@ -28,9 +28,11 @@
       "description": ".NET MAUI SDK Core Packages",
       "packs": [
           "Microsoft.Maui.Dependencies",
-          "Microsoft.Maui.Sdk",
+          "Microsoft.Maui.Sdk-@MAUI_DOTNET_VERSION@",
+          "Microsoft.Maui.Sdk-@MAUI_PREVIOUS_DOTNET_VERSION@",
```

Comment:
Instead of pulling in the pack directly, we pull in aliased versions for the current and previous releases.